features/admin.extensions.v1/components/password-history-count/components/password-history-count.tsx

```diff
@@ -16,11 +16,13 @@
  * under the License.
  */
 
+import Alert from "@oxygen-ui/react/Alert";
 import { Field } from "@wso2is/form/src";
-import { Message } from "@wso2is/react-components/src";
-import React, { ReactElement } from "react";
+import {
+    Heading
+} from "@wso2is/react-components";
```

Comment:
Let's only split lines when importing three or more.

```suggestion
import { Heading } from "@wso2is/react-components";
```

Reply:
Fixed